lab_5_scrapper/scrapper.py

```diff
@@ -120,45 +231,85 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
+        self.article = Article(self.full_url, self.article_id)
```

Comment:
again, are you sure all the attributes should be public here?

Reply:
Made config a private attribute